libres/lib/enkf/enkf_fs.cpp

```diff
@@ -243,12 +243,13 @@ enkf_fs_type *enkf_fs_alloc_empty(const char *mount_point) {
         if (util_try_lockf(fs->lock_file, S_IWUSR + S_IWGRP, &fs->lock_fd)) {
             fs->read_only = false;
         } else {
-            fprintf(stderr, " Another program has already opened filesystem "
-                            "read-write - this instance will be UNSYNCRONIZED "
-                            "read-only. Cross your fingers ....\n");
-            fs->read_only = true;
```

Comment:
`fs->read_only = true;` not needed anymore or just removed by accident when refactoring?

Reply:
This assignment was not needed even before the code change and I cleaned it up. It is assigned as true by default on line 223 so there was no reason to set it again.